plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -3764,24 +3766,24 @@ private Long restoreVolumeFromSecStorage(VmwareHypervisorHost hyperHost, Datasto
             throw new Exception(msg);
         }
 
-        VirtualMachineMO clonedVm = null;
+        VirtualMachineMO workerVm = null;
```

Comment:
```suggestion
        VirtualMachineMO importedVm = null;
```

Reply:
I feel `workerVm` suits more here as we use the VM just to import the OVF created for volume that is to be restored from sec store.

Reply:
@shwstppr If this is a worker VM, please tag it accordingly so that it can be tracked (when not destroyed, in failed scenarios) and cleaned up when worker vms cleanup is enabled.

Reply:
Done